test/auth.js

```diff
@@ -19,12 +20,18 @@ async function getAccessTokenHeaders ({
     editNamespaces
   })
 
-  return {
+  const headers = {
     authorization: `Bearer ${accessToken}`,
     // TODO: use an API key instead of these special headers
     'x-platform-id': t.context.platformId,
     'x-stelace-env': t.context.env
   }
+
+  if (apiVersion) {
+    headers['x-stelace-version'] = apiVersion
```

Comment:
For reference, can you explain why you need this?
It seems we can already ask for a specific pagination on a per-request basis.

Why do you need this to get access tokens?
Probably for SDK and pagination handling, but context would be useful to future reader anyway.

Reply:
Indeed, I'll add a comment for that.

The request header 'x-stelace-version' force the use of a specific API version even if the platform is in a different API version. Putting this parameter on `getAccessTokenHeaders` function is practical because it is used by almost all tests when setting API permissions.

By providing the parameter `apiVersion` in this function, tests can check that old behaviour (from older versions platforms) is still working. So the breaking change is correctly versioned.
In this case, I wanted to test that old platforms will work with offset pagination.

Reply:
Then your comment should just stay this allows to skip `x-stelace-version` header in individual requests, right?

Reply:
Yes indeed.

Reply:
Your added comment is confusing.

I wrote

> allows to skip `x-stelace-version` header in individual requests

because we already had this option before. What you want to do when including it in access token is to skip this header in each and every individual request using this token, right?
In this case this is not what your comment says.